xpdtools/tools.py

```diff
@@ -580,3 +580,11 @@ def inner(x, *args, **kwargs):
         return func(*args, **kwargs)
 
     return inner
+
+
+def decomp(data, variance=.9):
```

Comment:
this is only one way to decompose a matrix.....should we call this ``pca_decomp``?  Or probably more correct, ``pc_decomp``?

Reply:
I'm planning on setting up a kwarg to allow users to pass in a decomposition engine in so any system which works the same can be used. I did this for now for simplicity and to keep as close to example code as possible